Pareto/StatusBar/StatusBarController.swift

```diff
@@ -108,6 +108,11 @@ class StatusBarController: NSObject, NSMenuDelegate {
         }
         // invalidate possible expired cache
         try! AppInfo.versionStorage.removeExpiredObjects()
+        
+        // Clear UpdateService cache when user manually runs checks
+        if interactive {
+            UpdateService.shared.clearCache()
+        }
```

Comment:
We had this, removed since this caused updates calls too be too frequent. You want it back?

Reply:
That should be fine.